aliasmgr/aliasmgr.go

```diff
@@ -259,7 +279,7 @@ func (m *Manager) AddLocalAlias(alias, baseScid lnwire.ShortChannelID,
 		return aliasToBaseBucket.Put(aliasBytes[:], baseBytes[:])
 	}, func() {})
 	if err != nil {
-		return err
+		returnErr = err
```

Comment:
Should we really **not** return here?
We would update the internal state of the manager and get a mismatch between DB and in-memory view.

Reply:
Let me know if https://github.com/lightningnetwork/lnd/pull/8509#discussion_r1521311364 closes this